GrandCentralBoard/Extensions/String.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  String.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 11.05.2016.
+//  Copyright © 2016 Macoscope. All rights reserved.
+//
+
+import Foundation
+import MD5
+
+extension String {
+    func containsAnyString(strings: [String]) -> Bool {
+        for string in strings where containsString(string) {
+            return true
+        }
+        return false
+    }
+
+    func stringByRemovingOccurrencesOfStrings(strings: [String]) -> String {
+        var result: String = self
+        for string in strings {
+            result = result.stringByReplacingOccurrencesOfString(string, withString: "")
+        }
+        return result
+    }
+}
+
+extension String {
+    func md5Hash() -> String? {
```

Comment:
Is this needed in separate `extension`? We can put this method right below `stringByRemovingOccurrencesOfStrings`.